### PR TITLE
pay frequency type issue resolved

### PR DESCRIPTION
**Pull request by analoo**

### CODE REVIEW

File: src/main/resources/templates/laDigitalAssister/jobPayPeriod.html [hunk 1]
```diff
@@ -14,34 +14,27 @@
...
-                                    th:replace="~{fragments/inputs/radio :: radio(inputName='payPeriod',value=#{job-pay-period.twice-a-month}, label=#{job-pay-period.twice-a-month})}"/>
-                                <th:block
-                                    th:replace="~{fragments/inputs/radio :: radio(inputName='payPeriod',value=#{job-pay-period.every-month}, label=#{job-pay-period.every-month})}"/>
-                                <th:block
-                                    th:replace="~{fragments/inputs/radio :: radio(inputName='payPeriod',value=#{job-pay-period.it-varies}, label=#{job-pay-period.it-varies})}"/>
-                              </th:block>
-                            </th:block>
+                  <th:block th:ref="radioContent">
+                    <th:block
+                        th:each="payPeriod : ${T(org.ladocuploader.app.data.enums.PayFrequencyType).values()}">
```
Comment (sree-cfa) [inline src/main/resources/templates/laDigitalAssister/jobPayPeriod.html:27]: nice 👌

File: src/main/resources/templates/laDigitalAssister/jobPayAmount.html [hunk 2]
```diff
@@ -10,21 +11,27 @@
         <div th:replace="~{fragments/goBack :: goBackLink}"></div>
         <main id="content" role="main" class="form-card spacing-above-35">
           <p class="grey-text" th:text="${fieldData.get('employerName')}"></p>
+          <th:block th:with="payPeriod =${fieldData.get('payPeriod')}">
+            <th:block
```
Comment (sree-cfa) [inline src/main/resources/templates/laDigitalAssister/jobPayAmount.html:15]: seems like these blocks can be merged?

**Comment by sree-cfa** [comment]
@analoo  I'm getting a 500 when I try to change the language on the pay period screen

File: src/main/resources/templates/laDigitalAssister/jobPayAmount.html (hunk 2 above)
Comment (analoo) [reply src/main/resources/templates/laDigitalAssister/jobPayAmount.html:15]: I believe that's current behavior (just tested it in staging).

**Comment by analoo** [comment]
@sree-cfa - I believe that the 500 errors when changing messages are current behavior. I tested it in Staging.

**Comment by sree-cfa** [comment]
> @sree-cfa - I believe that the 500 errors when changing messages are current behavior. I tested it in Staging.

Oh i see

**Review by sree-cfa — APPROVED** [review]
lgtm

**Pull request merged**